Dockerfile

```diff
@@ -37,6 +37,20 @@ ADD clean-logs.sh /clean-logs.sh
 # Add startup script
 ADD virtuoso.sh /virtuoso.sh
 
+# Let's add some other dependencies
+RUN apt-get install -y python2.7 python-pip awscli
+
+ADD crontab /etc/cron.d/periodic_back_cron
+
+RUN chmod 0644 /etc/cron.d/periodic_back_cron
+
+#ADD periodic_backup.sh /periodic_backup.sh
+#RUN chmod u+x /periodic_backup.sh
+
+RUN touch /var/log/cron.d
```

Comment:
interesting - that makes sense - another similar issue with the logstash container comes to mind that ill see if this can help resolve. Thanks for the info 👍